controllers/glanceapi_controller.go

```diff
@@ -661,6 +694,11 @@ func (r *GlanceAPIReconciler) reconcileNormal(ctx context.Context, instance *gla
 			err.Error()))
 		return ctrlResult, err
 	} else if (ctrlResult != ctrl.Result{}) {
+		instance.Status.Conditions.MarkFalse(
```

Comment:
ditto this feels something in progress

Reply:
same comment I had above

Reply:
Comparing `ctrlResult != ctrl.Result{}` feels like something still `in-progress` rather than an `error` that should mark the `condition` to `False` and return an `err`.
I'll change this part as well in my next iteration to match the other similar comment I had in a different piece of code that does the same comparison.